13/umbraco-cms/fundamentals/setup/install/README.md

```diff
@@ -11,7 +11,15 @@ The fastest way to get the latest version of Umbraco up and running is using the
 1. Open your command line.
 2. Install the Umbraco templates:
 
-`dotnet new install Umbraco.Templates::13.*`
+```bash
+dotnet new install Umbraco.Templates::13.*`
+```
+
+e.g.
+
+```bash
+dotnet new install Umbraco.Templates::13.8.1`
+```
```

Comment:
How about we add a bit more explanation to it as well:

```suggestion
{% hint style="info" %}

Replace `13.*` with the specific version you want to install.

Example:
```bash
dotnet new install Umbraco.Templates::13.8.1`
```

{% endhint %}
```